TombEngine/Game/control/trigger.cpp

```diff
@@ -152,6 +152,10 @@ int SwitchTrigger(short itemNumber, short timer)
 	}
 	else if (item->Status)
 	{
+		if (item->ObjectNumber == ID_AIRLOCK_SWITCH &&
+			item->Animation.AnimNumber == GetAnimNumber (*item, 2) &&
+			item->Animation.FrameNumber == GetFrameNumber (item, 0))
+			return 1;
```

Comment:
Good opportunity to modify `SwitchTrigger()` to return a `bool` instead of 0 or 1.

Reply:
done

Reply:
I unmade this change, as Raildex asked, I can remake it on a future branch.

(Once the airlock and switches were merged as both makes some modifications to that function).